backend/src/server.js

```
@@ -303,22 +311,44 @@ async function startServer() {
     // Graceful shutdown
-    process.on('SIGTERM', () => {
-      logger.info('SIGTERM signal received: closing HTTP server');
-      server.close(async () => {
-        logger.info('HTTP server closed');
-        await closeConnection();
-        process.exit(0);
-      });
-    });
+    let isShuttingDown = false;
+    const shutdown = async (force = false) => {
+      if (isShuttingDown) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)